models/src/main/thrift/content/v1.thrift

```diff
@@ -377,6 +380,8 @@ struct AssetFields {
   68: optional bool safeEmbedCode
 
   69: optional bool isMandatory
+
+  70: optional list<CartoonVariant> variants
```

Comment:
Let's make this `cartoonVariants` as a generic `variants` may create a naming clash in the future

Reply:
https://github.com/guardian/content-api-models/pull/223/commits/1c37edd98d19e238a715a39b13be098a64bfbc33